client/src/components/Event.jsx

```diff
@@ -37,7 +37,8 @@ function Event ({props}) {
     
     return (
         <div className="event">
-            <a href="#"><div className="inner image" style={{ 'backgroundImage': `url("${props.image_url}")` }}></div></a>
+            {/* TODO: Change hardcoded URL to dynamic environmental variable */}
+            <a href="#"><div className="inner image" style={{ 'backgroundImage': `url("http://localhost:3016/${props.image_url}")` }}></div></a>
             <i className="far fa-bookmark fa-2x bookmark"
```

Comment:
Since we are not actually going to render a new page when someone clicks on an image, might we not want to 1href` attribute? When someone clicks on it, the user is brought all the way to the top.  This seems like good functionality if we had a "Back To Top" button.